common/models/user.js

```diff
@@ -382,6 +413,49 @@ module.exports = function(User) {
     }
   );
 
+  User.stats = function stats(username, cb) {
+    if (!username) {
+      return nextTick(() => {
+        cb(new TypeError(
+          `username should b a string but got ${ username }`
```

Comment:
Would it be useful to use a `typeof` aswell, just as an extra aid to debugging 

`but got ${ username }, of type ${ typeof username }`